packages/env-config/index.ts

```diff
@@ -0,0 +1,17 @@
+import { createEnv } from "@t3-oss/env-nextjs"
+import { z } from "zod"
+
+export const env = createEnv({
+  client: {
+    NEXT_PUBLIC_SOCKET_URL: z.string().default("ws://localhost:4001"),

```

Comment:
The right port is `4000`

```suggestion
    NEXT_PUBLIC_SOCKET_URL: z.string().default("ws://localhost:4000"),
```